js/tji-datasets-explore.js

```diff
@@ -476,9 +478,12 @@ TJIChartView.prototype.create_filters = function() {
 
 TJIChartView.prototype.create_filter_checkboxes = function(filter) {
   var that = this;
-  var fieldset = jQuery('<fieldset><legend>' + filter.name.replace(/_/g, " ") + '<i class="fas fa-caret-down"></i></legend></fieldset>');
+  var fieldset = jQuery('<fieldset><legend class="js-chart-legend">' + filter.name.replace(/_/g, " ") + '</legend></fieldset>');
```

Comment:
Hey @LayaTaal not sure where this code came from? The Filter collapse and expand isn't yet merged.